latex_access/tests/test_settings.py

```diff
@@ -0,0 +1,40 @@
+import unittest
+import latex_access
+from latex_access import settings as settings_module
+
+try:
+    settings_module.settings['brailletable'] = 'nemeth'
+    settings_module.brailleTableToUse()
+    settings_module.settings['brailletable'] = 'ueb'
+    settings_module.brailleTableToUse()
+    incompatible = False
+except AttributeError:
+    incompatible = True
+
+class TestSettings(unittest.TestCase):
+    def test_load_settings(self):
+        """Tests that settings are read from a file."""
+        settings_module.settings = {}
```

Comment:
Clearing default values from settings here is wrong. While you're adding a subset of values from file further down imagine that a different test (not necessarily for the settings module) also uses settings and fails  because you've removed the default values. You should:
- Create a copy of a default settings in a `setUp` method (take care to create a copy not just a reference)
- Restore them after each test in `tearDown`.